dspace/config/log4j.properties

```diff
@@ -22,7 +22,7 @@ log.dir=${dspace.dir}/log
 #   Log level for all DSpace-specific code (org.dspace.*)
 #   Possible values (from most to least info): DEBUG, INFO, WARN, ERROR, FATAL
 #   Defaults to INFO
-loglevel.dspace=INFO
+loglevel.dspace=DEBUG
```

Comment:
Please remove this accidental change. The default should be `INFO`

Reply:
done